docs/includes/apiargs-MongoDBDatabase-method-createCollection-option.yaml

```diff
@@ -133,6 +133,19 @@ interface: phpmethod
 operation: ~
 optional: true
 ---
+arg_name: option
+name: timeseries
+type: array|object
+description: |
+  Available on 5.0+ servers only
+
+  Allows users to specify options for time-series collections.
```

Comment:
Note: I haven't found additional documentation for this param publicly yet, so I've only stubbed out documentation. I suggest we revisit this closer to the 5.0 release cycle if at all. In that case, I'd create a follow-up ticket, but I'm also fine linking to the command documentation for more information.